cmd/k8s-operator/deploy/examples/proxyclass.yaml

```diff
@@ -15,3 +15,9 @@ spec:
         kubernetes.io/os: "linux"
       imagePullSecrets:
       - name: "foo"
+      tailscaleContainer:
+        image: "ghcr.io/tailscale/tailscale:v1.64.0"
+        imagePullPolicy: IfNotPresent
+      tailscaleInitContainer:
+        image: "ghcr.io/tailscale/tailscale:v1.64.0"
```

Comment:
nit: Maybe either update to the latest tag (1.66.4) as of today, or use stable?

Reply:
I'll put `1.64` in 👍🏼 This is just an example, but I generally find static image tags to be not a great practice